app/Http/Controllers/Store/CentreController.php

```diff
@@ -50,20 +52,51 @@ public function printCentreCollectionForm(Centre $centre)
     }
 
     /**
-     * Exports a summary of registrations from the User's relevant Centres.
+     * Exports a summary of registrations from the User's relevant Centres or specified Centre.
      *
-     * @return \Illuminate\Contracts\Routing\ResponseFactory|\Symfony\Component\HttpFoundation\Response
+     * @param Centre $centre
+     * @return ResponseFactory|\Symfony\Component\HttpFoundation\Response
      */
-    public function exportRegistrationsSummary()
+    public function exportRegistrationsSummary(Centre $centre)
     {
         // Get User
         $user = Auth::user();
 
-        // Get now()
-        $now = Carbon::now();
+        if (isset($centre->id)) {
+            // Set for specified centre
+            $centre_ids = [$centre->id];
+            $dateFormats = [
+                'dob' => 'm/Y'
+            ];
+        } else {
+            // Set for relevant centres
+            $centre_ids = $user->relevantCentres()->pluck('id')->all();
+            $dateFormats = [];
```

Comment:
Is this because multiple centres means it's an ARC user download - not one that needs pretty dates?

Reply:
Ah no - I see there are many date formats...

Reply:
`$dateFormats` defaults to `[]` do you need to pass it?

Reply:
I'd at least need to initialise it as null, as further down i just stuff the function with params.